frontend/src/app/page.tsx

```
@@ -0,0 +1,303 @@
+                  </h1>
+                  <p className="text-text-sub text-lg font-normal leading-relaxed max-w-lg">
+                    저장하고, 자동 태그하고, 다시 활용하세요.<br/>
+                    SearchWeb의 AI가 당신의 지식 관리를 돕습니다.<br/>
+                    단순한 링크 저장을 넘어선 지식 베이스를 구축하세요.
+                  </p>
+                </div>
+                
+                <div className="flex flex-col sm:flex-row gap-4 pt-2">
+                  <NextLink href="/login" className="flex items-center justify-center gap-2 bg-[linear-gradient(135deg,#6d28d9_0%,#8b5cf6_50%,#a78bfa_100%)] hover:brightness-105 text-white text-sm font-semibold h-11 px-6 rounded-lg transition-all shadow-[0_10px_25px_-5px_rgba(124,58,237,0.4),0_8px_10px_-6px_rgba(124,58,237,0.2)] whitespace-nowrap border-t border-white/20">
+                    <div className="flex h-5 w-5 items-center justify-center rounded-sm bg-white p-0.5">
+                      <svg width="14" height="14" viewBox="0 0 24 24">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)